test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -1,15 +1,31 @@
 package seedu.addressbook.common;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import org.junit.Test;
 
 import java.util.Arrays;
-import java.util.List;
 
-import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 public class UtilsTest {
 
+    @Test
+    public void isAnyNullTest() {
+        Utils testing = new Utils();
+
+        // Testing one null case
+        assertEquals(testing.isAnyNull((Object)null), true);
+
+        // Testing one object
+        assertEquals(testing.isAnyNull("Hello"), false);
+
```

Comment:
Its better to follow similar conventions of testing pattern to other code in the project. In the example test method provided (elementsAreUnique), we used assertFalse and assertTrue for testing. Here, it seems redundant to use assertEquals for True/False assertion instead of using readily available assertTrue/assertFalse.